changes/changes-8.8.0.md

```diff
@@ -41,6 +41,7 @@ Release Notes.
 * Optimize okhttp 3.x 4.x plugin to get span time cost precisely
 * Adapt message header properties of RocketMQ 4.9.x
 * Fix httpasyncclient-4.x-plugin's memory leak risk
+* Fix the bug that httpasyncclient-4.x-plugin  puts the dirty tracing context in the connection context
```

Comment:
I think you missed a comment before. I mean this change log should be added into the `changes.md` in the root, not 8.8.0. The 8.8.0 is a released version, which should not have a new change anymore.